drop-transfer/src/error.rs

```diff
@@ -116,6 +116,6 @@ impl From<walkdir::Error> for Error {
         value
             .into_io_error()
             .map(Into::into)
-            .unwrap_or(Error::BadPath)
+            .unwrap_or(Error::BadPath("Filesystem loop detected".into()))
```

Comment:
Loop?

Reply:
```rust
#[derive(Debug)]
enum ErrorInner {
    Io { path: Option<PathBuf>, err: io::Error },
    Loop { ancestor: PathBuf, child: PathBuf },
}
```
that's the `walkdir` error, so if it's not IO then it's loop (probably should never happen as we check for the links)

Reply:
Makes sense, thanks